css-inline/src/main.rs

```diff
@@ -199,6 +201,9 @@ OPTIONS:
 
     --keep-link-tags
         Keep "link" tags after inlining.
+        
```

Comment:
There is a small place above where this flag should be added too. The handle_boolean_flag function